libparsec/crates/client_connection/tests/sse.rs

```diff
@@ -0,0 +1,114 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+#![cfg(not(target_arch = "wasm32"))]
+
+mod proxy;
+
+use futures_util::StreamExt;
```

Comment:
can you use `libparsec_platform_async::stream::StreamExt` (i.e. `futures_lite::stream::StreamExt`) instead ?

Reply:
done